### PR TITLE
Fix reclaim mannequin command without CSV

### LINKED ISSUE #760: `gh gei reclaim-mannequin`'s single user (not CSV) flow does not work
## Description

The `gh gei reclaim-mannequin`'s documented option to specify a `--mannequin-user` and `--target-user` does not work.

Even when both arguments are specified, you get the following error:

```
[11:50] [ERROR] Either --csv or --mannequin-user and --target-user must be specified
```

Specifying `--csv` works as expected.

## Reproduction Steps

Run `gh gei reclaim-mannequin --github-target-org Deep-Avocado-9 --mannequin-user foo --target-user foo` on `gh-gei` v0.31.

### PR DESCRIPTION
**Pull request by tspascoal**
Individual reclaiming of a mannequin wasn't working. mannequin and target user were not being passed.

closes #760 

- [ ] Did you write/update appropriate tests
- [x] Release notes updated (if appropriate)
- [ ] Appropriate logging output
- [x] Issue linked
- [ ] Docs updated (or issue created)

### CODE REVIEW

**Review by dylan-smith — APPROVED** [review]

File: RELEASENOTES.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 - Added logic to check if a target repo exists before generating GHES archives
-- Added logic to ensure target org exists before generating GHES archives
-
+- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were were not being passed to the reclaim command
```
Comment (ArinGhazarian) [inline RELEASENOTES.md:2]:
```suggestion
- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command
```

**Review by ArinGhazarian — APPROVED** [review]

File: RELEASENOTES.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 - Added logic to check if a target repo exists before generating GHES archives
-- Added logic to ensure target org exists before generating GHES archives
-
+- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command
```
Comment (timrogers) [inline RELEASENOTES.md:2]: I suggest mentioning the command, enclosing the parameters in backticks and dropping the explanation of why it was broken (which only really matters for us!) :heart:

```suggestion
- Fixed reclaiming a single mannequin using `reclaim-mannequin` with the `--mannequin-user` and `--target-user` parameters
```

**Review by timrogers — CHANGES_REQUESTED** [review]
Thank you so much for taking the time to fix this ❤️

File: RELEASENOTES.md (hunk 2 above)
Comment (tspascoal) [reply RELEASENOTES.md]: @timrogers done

**Review by timrogers — APPROVED** [review]

**Pull request merged**